docroot/themes/humsci/humsci_basic/templates/views/views-view-grid.html.twig

```diff
@@ -0,0 +1,40 @@
+{#
+/**
+ * @file
+ * Default theme implementation for views to display rows in a grid.
+ *
+ * Available variables:
+ * - attributes: HTML attributes for the wrapping element.
+ * - title: The title of this group of rows.
+ * - view: The view object.
+ * - rows: The rendered view results.
+ * - options: The view plugin style options.
+ *   - row_class_default: A flag indicating whether default classes should be
+ *     used on rows.
+ *   - col_class_default: A flag indicating whether default classes should be
+ *     used on columns.
+ * - items: A list of grid items. Each item contains a list of rows or columns.
+ *   The order in what comes first (row or column) depends on which alignment
+ *   type is chosen (horizontal or vertical).
+ *   - attributes: HTML attributes for each row or column.
+ *   - content: A list of columns or rows. Each row or column contains:
+ *     - attributes: HTML attributes for each row or column.
+ *     - content: The row or column contents.
+ *
+ * @see template_preprocess_views_view_grid()
+ *
+ * @ingroup themeable
+ */
+#}
+{% set viewItems = [] %}
+{% if title %}
+  <h3>{{ title }}</h3>
+{% endif %}
+
+{% for row in items %}
+  {% for column in row.content %}
+      {% set viewItems = viewItems|merge([column.content]) %}
+  {% endfor %}
+{% endfor %}
+
+{{ pattern('grid', {items: viewItems, columns: options.columns}) }}
```

Comment:
@nrambeck, I see the `grid` UI pattern being used but not `grid_two_column` or `grid_three_column`. Are those UI patterns earlier iterations that can be deleted from this PR?

Reply:
yes, @bryanbraun we started yesterday thinking the two and three column ui patterns could be used, but it looks like just the Grid one is used one.

Reply:
I can make a commit with those changes